apps/backend/src/modules/users/user.entity.ts

```diff
@@ -92,20 +92,16 @@ class UserEntity implements Entity {
 	}
 
 	public toNewObject(): {
-		createdAt: string;
```

Comment:
Shouldn't be removed

Reply:
Not fixed

Reply:
Fields createdAt, updatedAt removed from code in 69f06c787fc2473d7c6fd4c44f2df4ff7a56d4bc Commit

Reply:
I see, that they are removed, you should add them back